fsh-generated/resources/StructureDefinition-au-erequesting-path-request.json

```diff
@@ -0,0 +1,42 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "au-erequesting-path-request",
```

Comment:
Apply the changes to this file from the other comments e.g. id, filename, name, description, title.